src/components/ModalWrapper.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+class ModalWrapper extends PureComponent {
+  // TODO remove mock styling
+  render() {
+    const { children } = this.props;
+    return (
+      <div style={{
+        width: '500px', height: '500px', position: 'fixed', left: '400px', top: '400px', backgroundColor: 'red',
+      }}>
```

Comment:
Make 50% transparent overlay and center content vertically and horizontally.
Add close action to the overlay.